lib/router/router.js

```diff
@@ -127,16 +135,21 @@ export default class Router extends EventEmitter {
       data, props, error
     } = await this.getRouteInfo(route, pathname, query)
 
-    if (error) {
+    if (error && error.cancelled) {
       this.emit('routeChangeError', error, as)
-      throw error
+      return false
     }
```

Comment:
You mean we notify nothing if it's aborted ? Anyway it should be another issue, I don't want to change the behavior on this PR.